GG/src/dialogs/FileDlg.cpp

```diff
@@ -216,6 +216,18 @@ void FileDlg::SetFileFilters(const std::vector<std::pair<std::string, std::strin
 const fs::path& FileDlg::WorkingDirectory()
 { return s_working_dir; }
 
+const boost::filesystem::path FileDlg::StringToPath(const std::string& str) {
+    #if defined(FREEORION_WIN32)
```

Comment:
Need to deal with raw strings anyways, since we read part of the path from the text. Since need the UTF logic anyways, I thought best to make the interface simpler and avoid including boost stuff in the header.